Makefile

```diff
@@ -0,0 +1,9 @@
+all:
```

Comment:
You can use a single rule which forwards every possible target to the `src/` subdirectory:

```make
$(MAKECMDGOALS) all:
	@$(MAKE) -C src $@

.PHONY: $(MAKECMDGOALS) all
```